### PR TITLE
Add Kubernetes agent GA blog post

### PR DESCRIPTION
**Pull request by APErebus**
# About

The Kubernetes agent moving from EAP to GA (general availability)

# Release

Current cloud

# Blog image idea

We can probably just re-use the same image as the kubernetes agent announcement post

# Before you submit your post

Check everything off this list before you submit your post for review. 

- [x] I'm familiar with the [Octopus voice and style](https://www.octopus.design/932c0f1a9/p/13df3a-voice-and-style), the [guide for blog content basics](https://www.octopus.design/932c0f1a9/p/901d2a-blog-content), and [writing educational content](https://www.octopus.design/932c0f1a9/p/137b06-educational-content---how-we-write-about-what-we-do).
- [ ] I've checked the content using Grammarly (Octopus has a corporate license) and the [Hemingway app](https://hemingwayapp.com/) to remove typos, grammatical errors, and passive voice. 
- [ ] The PR build passes validation, and if it doesn't, I've checked the [common validations errors](https://www.octopus.design/932c0f1a9/p/901d2a-blog-content/t/817249) and none of those apply.
- [ ] The images I've included follow the [rules for working with images](https://www.octopus.design/932c0f1a9/p/5061d7-working-with-images).
- [ ] I've added alt text for all the images I've included (alt text describes an image to people unable to see it, 125 characters max)
- [x] The draft is complete and this post is ready to be reviewed. (Please mark the post 'ready for review' at the bottom of the PR.)
- [x] I've dropped a note in Slack in #topic-content-marketing to let Tegan Ali know my post is ready for review. (No need to fill out a content brief.)
- [ ] I'll draft social media posts for Twitter and LinkedIn. I'll post them for review in #topic-social-media after the post has been edited or after the PR has been merged.

## How to review this PR

If there's anything you'd like reviewers to focus on, mention it here.

## Publication date

If there are considerations for when to publish this post, mention them here. For example, this post is supporting material for a webinar I'll be conducting on date, or this post should not published until after a specific release on date.

### CODE REVIEW

File: blog/2024-07/kubernetes-agent-ga/index.md [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+bannerImageAlt: Octopus Deploy logo connected to a Kubernetes-branded box with an Octopus logo in it.
+isFeatured: false
+tags: 
+  - Product
+  - Kubernetes
+---
+
+We released the Kubernetes agent in Octopus Server 2024.2 as an early access preview (EAP). While we were very confident in the functionality and stability of the agent, we were also aware of the variations in Kubernetes deployments and infrastructure. These variations can impact installation, execution, and stability of the agent.
+
+We had excellent uptake of the agent during the early access period. More than 25 customers used the agent and over half of them used it for production deployments. These customers deployed more than 225,000 times using the agent.
```
Comment (TeganAli) [inline blog/2024-07/kubernetes-agent-ga/index.md:18]: @APErebus is 25 customers correct? @nikita-dergilev mentioned 85 in the newsletter article, I think.

Comment (APErebus) [reply blog/2024-07/kubernetes-agent-ga/index.md:18]: @TeganAli hrm, let me double check that number. I pulled it out of amplitude, but maybe I was looking in the wrong spot

Comment (APErebus) [reply blog/2024-07/kubernetes-agent-ga/index.md:18]: @TeganAli I checked with Kit. The 25 refers to customers deploying regularly with the agent, where as the 85 is _all_ customers who've installed the agent (but maybe not deployed with it, or only did one deployment).

I'd like to keep with 25 in the blog post, but maybe we tweak it to say `More than 25 customers are deploying regularly with the agent and over half ...`

Comment (TeganAli) [reply blog/2024-07/kubernetes-agent-ga/index.md:18]: No worries. Thanks @APErebus . I'll make that change.

**Review by TeganAli — APPROVED** [review]

File: blog/2024-07/kubernetes-agent-ga/index.md [hunk 2]
```diff
@@ -0,0 +1,74 @@
+---
+title: Kubernetes agent general availability
```
Comment (TeganAli) [inline blog/2024-07/kubernetes-agent-ga/index.md:2]:
```suggestion
title: Kubernetes agent now generally available
```

**Review by TeganAli — APPROVED** [review]

**Pull request merged**